python/cudf/cudf/core/buffer/buffer.py

```diff
@@ -70,6 +72,95 @@ def cuda_array_interface_wrapper(
     )
 
 
+class BufferWeakref(object):
```

Comment:
Explicitly inheriting from `object` is not necessary.
```suggestion
class BufferWeakref:
```

Reply:
Done